testing/trino-server-dev/etc/catalog/iceberg.properties

```diff
@@ -12,5 +12,8 @@ connector.name=iceberg
 hive.metastore.uri=thrift://localhost:9083
 hive.hdfs.socks-proxy=localhost:1180
 
+# Ensure test retries don't write to non-empty locations
```

Comment:
What is the reasoning behind adding this setting for the DevelopmentServer Iceberg connector?

Reply:
See my comment https://github.com/trinodb/trino/pull/12227#issuecomment-1160422690

We can never really ensure that all cleanup (finally stuff) is run eg. in case of Hive timeouts that any table location will be emptied, so I think this is the best way to handle this as this ensures every table will have its unique location. This can be moved to a separate PR if necessary.